src/record/record-transition.js

```diff
@@ -147,6 +148,11 @@ RecordTransition.prototype.add = function (socketWrapper, version, message) {
       return
     }
 
+    if(!utils.isOfType(data, 'object') && !utils.isOfType(data, 'array')) {
```

Comment:
We use static references in the code base to types:

https://github.com/deepstreamIO/deepstream.io/blob/f69434acfbaeb21269714a927f3ffc47cdb37625/src/utils/utils.js#L5

Wondering if we should move this into a central place so that we can reference them, or maybe even have a 

`
utils.isOfTypeObject(data)
utils.isOfTypeArray(data)
`